Jenkinsfile

```diff
@@ -69,6 +71,18 @@ for(i = 0; i < buildTypes.size(); i++) {
 builds.failFast = failFast
 parallel builds
 
+if (buildTypes.contains("linux")) {
+    node("linux") {
+        unstash "metadata"
+        unarchive mapping: ['war/target/linux-jenkins.war': 'jenkins.war']
+        def fileUrl = "file://" + pwd() + "/jenkins.war"
```

Comment:
This will not be a valid URL in general. (Long list of reasons.) In the absence of a step doing a `File` → `URL` conversion, maybe the `jenkins` parameter can be made to accept a local file path?

Reply:
@jglick Can you elaborate on this? Just trying to understand because all my tests with this kind of URI worked perfectly fine with CURL, the ATH is actually always run against a local file, CURL is used to copy the file to the appropriate path in case of a remote downloadable URL or a local file URI like this case